public/style/main.css

```diff
@@ -128,6 +164,127 @@ nav div.section {
   overflow-y       : hidden;
   overflow-x       : hidden;
 }
+
+/* BOTTOM-UP CONTROL PANEL MODULES */
+
+.controlModules {
+  font-size: 1.2em;
```

Comment:
agree that font sizing assignment need to be more consistent.  but this is somewhat inevitable because we are still iterating the layout and testing out what's the right sizings of things are.